packages/paste-core/components/button/src/styles.ts

```diff
@@ -10,7 +10,7 @@ import {ButtonWrapperProps, ButtonChildrenProps} from './types';
  */
 const sizeReset = css`
   font-size: 100%;
-  padding: 0;
+  padding: 2px;
```

Comment:
No size token for this. We need padding here because removing the border makes the text right next to the edge of the background.